commons/src/main/kotlin/com/simplemobiletools/commons/adapters/MyRecyclerViewListAdapter.kt

```diff
@@ -99,7 +99,7 @@ abstract class MyRecyclerViewListAdapter<T>(
 
                 if (baseConfig.isUsingSystemTheme) {
                     actBarTextView?.onGlobalLayout {
-                        val backArrow = activity.findViewById<ImageView>(R.id.action_mode_close_button)
+                        val backArrow = activity.findViewById<ImageView>(androidx.appcompat.R.id.action_mode_close_button)
```

Comment:
Could this be Material or AppCompat or another, I wasn't sure which one the synthetics was importing